docker_images/diffusers/app/lora.py

```diff
@@ -64,15 +66,25 @@ def _load_lora_adapter(self, kwargs):
                     self.current_lora_adapter,
                     adapter,
                 )
-                # self.ldm.unfuse_lora()
+                if self.fused_lora:
+                    self.ldm.unfuse_lora()
                 self.ldm.unload_lora_weights()
                 self.current_lora_adapter = None
                 logger.info("LoRA weights unloaded, loading new weights")
                 weight_name = self._get_lora_weight_name(model_data=model_data)
                 self.ldm.load_lora_weights(
                     adapter, weight_name=weight_name, use_auth_token=self.use_auth_token
                 )
-                # self.ldm.fuse_lora()
+                try:
+                    self.ldm.fuse_lora(safe_fusing=True)
+                    self.fused_lora = True
+                except ValueError as e:
+                    logger.exception(e)
+                    logger.warning(
+                        "Unable to fuse lora weights for adapter %s", adapter
+                    )
```

Comment:
I would probably also unload the LoRA weights here

Reply:
done :)